ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionLogUpdatesConversions.scala

```diff
@@ -74,7 +78,8 @@ private[events] object TransactionLogUpdatesConversions {
               Seq(
                 FlatTransaction(
                   transactionId = transaction.transactionId,
-                  commandId = getCommandId(transaction.events, requestingParties),
+                  commandId = transaction.commandId
+                    .getOrElse(getCommandId(transaction.events, requestingParties)),
```

Comment:
Actually this getOrElse would never be called since it is populated with Some(commandId) in the filter method.

Reply:
you are right